core/src/main/scala/org/apache/spark/HeartbeatReceiver.scala

```diff
@@ -80,7 +80,9 @@ private[spark] class HeartbeatReceiver(sc: SparkContext, clock: Clock)
   // executor ID -> timestamp of when the last heartbeat from this executor was received
   private val executorLastSeen = new HashMap[String, Long]
 
-  private val executorTimeoutMs = sc.conf.get(config.STORAGE_BLOCKMANAGER_HEARTBEAT_TIMEOUT)
+  private val executorTimeoutMs = sc.conf.get(
+    config.STORAGE_BLOCKMANAGER_HEARTBEAT_TIMEOUT
+  ).getOrElse(Utils.timeStringAsMs(s"${sc.conf.get(Network.NETWORK_TIMEOUT)}s"))
```

Comment:
There are two reasons for change from 
```
sc.conf.get(
  config.STORAGE_BLOCKMANAGER_HEARTBEAT_TIMEOUT
).getOrElse(sc.conf.getTimeAsMs(Network.NETWORK_TIMEOUT.key))
```
to 

```
sc.conf.get(
    config.STORAGE_BLOCKMANAGER_HEARTBEAT_TIMEOUT
  ).getOrElse(Utils.timeStringAsMs(s"${sc.conf.get(Network.NETWORK_TIMEOUT)}s"))
```
：

- Maybe `Network.NETWORK_TIMEOUT` is not configured, need a default value

- TimeUnit of `Network.NETWORK_TIMEOUT` is `TimeUnit.SECONDS` , so if user configure  `Network.NETWORK_TIMEOUT`  to `200` not `200s`, the result of `executorTimeoutMs` will be `200ms` not `200000ms` when use `sc.conf.getTimeAsMs(Network.NETWORK_TIMEOUT.key)`, it should be wrong

Reply:
Yeah, that's fine.